examples/custom_filter_example.py

```diff
@@ -0,0 +1,125 @@
+from typing import Any, Union
+from aiogram import Bot, Dispatcher, executor, types
+from aiogram.dispatcher.filters import BoundFilter
+from aiogram.dispatcher.handler import ctx_data
+from aiogram.dispatcher.middlewares import LifetimeControllerMiddleware
+
+API_TOKEN = "BOT_TOKEN_HERE"
+
+
+fake_db = {
+    0000000000: {"ban": True},
+    1111111111: {"ban": False},
+    2222222222: {"ban": True},
+    3333333333: {"ban": False},
+}
+
+
+bot = Bot(token=API_TOKEN)
+dp = Dispatcher(bot)
+
+
+class UserMiddleware(LifetimeControllerMiddleware):
```

Comment:
Should we isolate topics: filters / middleware?
Or is a complex example better?

\+ @aiogram/dev-team

Reply:
I think working with the context can be taken as a separate example